crates/holochain_integrity_types/src/capability/grant.rs

```diff
@@ -122,6 +123,7 @@ impl CapGrant {
 
 /// Represents access requirements for capability grants.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, Hash)]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
 pub enum CapAccess {
```

Comment:
This one is an input type, so this one makes sense and then we'd need to define another type for a list operation to not include the secrets

Reply:
Don't think this is relevant any more, because a separate type got defined